README

```diff
@@ -127,6 +127,10 @@ interactive::
 Set to prompt a message and wait before testing the presence of a U2F device.
 Recommended if your device doesn't have a tactile trigger.
 
+[prompt=your prompt here without newline]::
+Set individual prompt message for interactive mode. Watch the square brackets
+around this parameter to get spaces correct recognized by PAM.
```

Comment:
nit: `correct` -> correctly
Also, why do you need brackets to ensure proper spacing? Is that a PAM thing?